.github/workflows/maven-deploy-fido2-tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Deploy FIDO2 Tests with Maven
```

Comment:
We build two WARs of PAS by a single action. But why not, we may split it here. It is probably better to have such granularity.

Reply:
The problem are other modules so we need to call only build of test server and fido2 tests.
And we cannot do it in one action as the resulting artifacts will clash on name.